test/user_test.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+import responses
+
+from matrix_client.api import MATRIX_V2_API_PATH
+from matrix_client.client import MatrixClient
+from matrix_client.user import User
+
+HOSTNAME = "http://localhost"
+
+
+class TestUser:
+    cli = MatrixClient(HOSTNAME)
+    user_id = "@test:localhost"
+    room_id = "!test:localhost"
+
+    @pytest.fixture()
+    def user(self):
+        return User(self.cli.api, self.user_id)
+
+    @pytest.fixture()
+    def room(self):
+        return self.cli._mkroom(self.room_id)
+
+    @responses.activate
+    def test_get_display_name(self, user, room):
```

Comment:
In future, please try to break tests that are testing multiple things into multiple methods.